services/generic/barrier_control.md

```diff
@@ -1,36 +1,94 @@
-
-Name        | Value example | Description
-------------|---------------|-------------
-`stopped_at`| 30            | Stopped at exact position (percentage value).
-
-#### Service props
-
-Name             | Value example                  | Description
------------------|--------------------------------|-------------
-`sup_notiftypes` | audio, visual                  | supported notification-types, like siren, flashlight
-`sup_states`     | open, closed, closing, opening | supported states
-`sup_tstates`    | open, closed                    | supported target states
\ No newline at end of file
+## Interfaces
+
+| Type | Interface                | Value type | Properties   | Description                                                                                                                                                                |
+|------|--------------------------|------------|--------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| in   | cmd.notiftype.get_report | null       |              | Requests notification types the device is using while opening/closing door.                                                                                                |
+| in   | cmd.notiftype.set        | bool_map   |              | Sets notification type the device is is using while opening/closing door. Configurable notification types are defined in [`sup_notiftypes`](#service-properties) property. |
+| out  | evt.notiftype.report     | bool_map   |              | Returns notification types the device is using while opening/closing door.                                                                                                 |
+| -    |                          |            |              |                                                                                                                                                                            |
+| in   | cmd.op.stop              | null       |              | Commence emergency stop of any operation.                                                                                                                                  |
+| -    |                          |            |              |                                                                                                                                                                            |
+| in   | cmd.state.get_report     | null       |              | Requests the current state of the device.                                                                                                                                  |
+| out  | evt.state.report         | string     | `stopped_at` | Reports the current state of the device, one of values defined in [`sup_states`](#service-properties) property.                                                            |
```

Comment:
'stopped_at' change to 'position' or 'cur_pos'. See discussion at https://github.com/futurehomeno/fimp-api/pull/71

Reply:
It's `stopped_at` because it is only sent when device is in `stopped` state.

Reply:
Maybe if Zigbee ad is able to use it also within other state's reports, e.g. when device is opening, then we should permit it? But why not use a typical out_lvl_switch then for that kind of functionalitites?

Reply:
I'd leave it for another discussion. If zigbee can be position aware in all times I'd suggest to either include dedicated interface for level reporting and leaving "stopped_at" as is. But also renaming it to "cur_pos" and changing it in zw implementation is an option too. Putting it on the post mortem list. Leaving as it is now.